docs/ug/configFiles.md

```diff
@@ -40,9 +40,9 @@ Given below are the details of the various config files used by RepoSense.
 | Ignore Commits List<sup>*+</sup> | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash. Additionally, a range of commits can be specified using the `..` notation e.g. `abc123..def456` (both inclusive). |
 | Ignore Authors List<sup>*+</sup> | The list of authors to ignore during analysis. Authors should be specified by their [Git Author Name](#a-note-about-git-author-name). |
 | Shallow Cloning | Enter **`yes`** to clone the repository using Git's shallow cloning functionality. This option can significantly reduce the time taken to clone large repositories. However, the option should ideally be disabled for smaller repositories where the `.git` file is smaller than 500 MB, as it would create overhead. |
-| File Size Limit<sup>+</sup>  | Enter a file size limit for the repository in bytes as a single number without units, that will override the default file size limit. |
+| File Size Limit<sup>+</sup> | Enter a file size limit for the repository in bytes as a single number without units (for a size limit of 1MB for example, enter 1000000). This file size limit will override the default file size limit (500KB). If file analysis is not skipped, files exceeding the file size limit will be marked as ignored and only the file name and line count will be reflected in the report. |
```

Comment:
Perhaps omit `If file analysis is not skipped,`? The skipping is explained further down. Mentioning it here (while it makes the explanation more accurate), might puzzle the reader.